Core/Object Arts/Dolphin/System/STON/STON.Writer.cls

```diff
@@ -178,7 +178,7 @@ jsonMode
 keepNewLines: boolean
 	"If true, any newline CR, LF or CRLF inside strings or symbols will not be escaped
 	but will instead be converted to the newline convention chosen, see #newLine:
-	The default is false, where CR, LF or CRLF will be enscaped unchanged."
+	The default is false, where CR, LF or CRLF will be escaped unchanged."
```

Comment:
"... will be escaped."

Reply:
Makes sense now I think about it. Okay, done.